capplets/display/display-capplet.ui

```diff
@@ -74,12 +82,12 @@
             </child>
             <child>
               <object class="GtkButton" id="apply_button">
-                <property name="label">gtk-apply</property>
+                <property name="label" translatable="yes">_Apply</property>
                 <property name="visible">True</property>
                 <property name="can_focus">True</property>
                 <property name="can_default">True</property>
                 <property name="receives_default">False</property>
-                <property name="use_stock">True</property>
+                <property name="use_underline">True</property>
```

Comment:
doesn't this loose the image on this button?

Reply:
It's recommended to replace the stock icon with only the label.
https://developer.gnome.org/gtk3/stable/gtk3-Stock-Items.html#GTK-STOCK-APPLY:CAPS

Reply:
Fair enough, but note that the neighbor buttons do have an icon.

Reply:
Before

![Captura de pantalla a 2019-12-23 11-03-55](https://user-images.githubusercontent.com/10171411/71351601-68a06e80-2574-11ea-87d1-0fdc42283c97.png)

After

![Captura de pantalla a 2019-12-23 11-06-34](https://user-images.githubusercontent.com/10171411/71351617-6fc77c80-2574-11ea-828d-f661654ae56e.png)

Reply:
OK, while different indeed, I would argue the new one actually looks better because of the consistency with the *Apply system-wide* button.